src/Iterator/AMQPMessageIterator.php

```diff
@@ -80,7 +83,7 @@ public function key(): void
         $this->counter;
     }
 
-    public function valid()
+    public function valid(): bool
```

Comment:
isnt this a bc break? or the class is final?

Reply:
the class is final